sesman/config.c

```diff
@@ -235,16 +236,22 @@ config_read_security(int file, struct config_security *sc,
             }
         }
         if (0 == g_strcasecmp(buf, SESMAN_CFG_SEC_ALWAYSGROUPCHECK))
-        {
-            sc->ts_always_group_check = g_text2bool((char *)list_get_item(param_v, i));
-        }
+		{
```

Comment:
bad indentation, please user spaces, not tabs.
look at xrdp coding style guide in wiki.